codegen/render_workspace.m

```diff
@@ -261,3 +404,14 @@ function write_mat(f, mat, name)
 fprintf(f, '%d};\n', mat.nz);
 
 end
+
+function write_mat_extern(f, mat, name)
+%WRITE_MAT_EXTERN Write matrix the prototype for the matrix.
+
+write_vec_extern(f, mat.i, [name, '_i'], 'c_int');
+write_vec_extern(f, mat.p, [name, '_p'], 'c_int');
+write_vec_extern(f, mat.x, [name, '_x'], 'c_float');
```

Comment:
I removed the three lines above in https://github.com/oxfordcontrol/osqp-matlab/commit/cee47d25218695e1c0b5390f7c7c8330d387f1b9